kubectl-minio/cmd/resources/common.go

```diff
@@ -107,9 +107,29 @@ func Pool(opts *TenantOptions, volumes int32, q resource.Quantity) miniov2.Pool
 	return p
 }
 
+func contains(s []string, t string) bool {
```

Comment:
With go 1.21.x we can have something like
```
import  "slices"

things := []string{"foo", "bar", "baz"}
slices.Contains(things, "foo") // true
```

Reply:
Resolved